frontend/packages/console-shared/src/components/modals/DeleteResourceModal.tsx

```diff
@@ -16,6 +16,7 @@ import { YellowExclamationTriangleIcon } from '../status';
 type DeleteResourceModalProps = {
   resourceName: string;
   resourceType: string;
+  actionLabel?: string;
   actionLabelKey?: string;
```

Comment:
Can we add a comment here explaining the difference between actionLabel and actionLabelKey?

Reply:
Added a comment for both props.